tests/cursor/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -251,8 +251,8 @@ const y = 5
 <|>
   
 =====================================output=====================================
-const y = 5;
-<|>
+const y = 5<|>;
+
```

Comment:
Looks like broken

Reply:
Yes, per the description, I can't find any way to fix performance without slightly changing the behavior. Given that formatWithCursor can currently churn in the `diff` method for >5s, I think it's worth the tradeoff, but it's up to the maintainers to decide.

Reply:
IMO as long as it works in most cases, it’s fine. Especially since the user can easily click back to where they were before.